examples/examples_test.go

```diff
@@ -20,41 +20,39 @@ import (
 	"testing"
 
 	"github.com/pulumi/pulumi/pkg/testing/integration"
-	"github.com/stretchr/testify/assert"
 )
 
-func TestDomain(t *testing.T) {
-	token := os.Getenv("NEWRELIC_API_KEY")
-	if token == "" {
-		t.Skipf("Skipping test due to missing NEWRELIC_API_KEY environment variable")
-	}
+func TestAccAlertPolicy(t *testing.T) {
+	test := getJSBaseOptions(t).
+		With(integration.ProgramTestOptions{
+			Dir: path.Join(getCwd(t), "alertpolicy"),
+		})
+
+	integration.ProgramTest(t, &test)
+}
 
+func getCwd(t *testing.T) string {
 	cwd, err := os.Getwd()
-	if !assert.NoError(t, err) {
-		t.FailNow()
+	if err != nil {
+		t.Error("expected a valid working directory", err)
```

Comment:
Will `Error` fail the test promptly? We used to get that behavior from `FailNow`.

Reply:
(same question on all of these PRs, I think)